.github/workflows/serenity-js-artifacts.yml

```diff
@@ -0,0 +1,105 @@
+
+    steps:
+      - name: Checkout SerenityOS/serenity
+        uses: actions/checkout@v3
+
+      - name: Install dependencies Ubuntu
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ninja-build unzip gcc-12 g++-12
+        if: ${{ matrix.os == 'ubuntu-22.04' }}
+
+      - name: Install dependencies macOS
+        run: |
+          brew install bash ninja unzip
+        if: ${{ matrix.os == 'macos-12' }}
+
+      - name: Check versions Ubuntu
+        run: |
+          ninja --version; gcc-12 --version; g++-12 --version
+        if: ${{ matrix.os == 'ubuntu-22.04' }}
+
+      - name: Check versions macOS
+        run: |
+          ninja --version; clang++ --version
+        if: ${{ matrix.os == 'macos-12' }}
+
+      - name: Create build directory
+        run: |
+          mkdir -p Build/TZDB
+          mkdir -p Build/UCD
+          mkdir -p Build/CLDR
+
+      - name: TimeZoneData cache
+        # TODO: Change the version to the released version when https://github.com/actions/cache/pull/489 (or 571) is merged.
+        uses: actions/cache@03e00da99d75a2204924908e1cca7902cafce66b
+        with:
+          path: ${{ github.workspace }}/libjs-test262/Build/TZDB
+          key: TimeZoneData-${{ hashFiles('Meta/CMake/time_zone_data.cmake') }}
+
+      - name: UnicodeData cache
+        # TODO: Change the version to the released version when https://github.com/actions/cache/pull/489 (or 571) is merged.
+        uses: actions/cache@03e00da99d75a2204924908e1cca7902cafce66b
+        with:
+          path: ${{ github.workspace }}/libjs-test262/Build/UCD
+          key: UnicodeData-${{ hashFiles('Meta/CMake/unicode_data.cmake') }}
+
+      - name: UnicodeLocale cache
+        # TODO: Change the version to the released version when https://github.com/actions/cache/pull/489 (or 571) is merged.
+        uses: actions/cache@03e00da99d75a2204924908e1cca7902cafce66b
+        with:
+          path: ${{ github.workspace }}/libjs-test262/Build/CLDR
+          key: UnicodeData-${{ hashFiles('Meta/CMake/locale_data.cmake') }}
+
+      - name: Create build directory Ubuntu
+        run: |
+          cmake -S Meta/Lagom -B Build -G Ninja \
+            -DCMAKE_C_COMPILER=gcc-12 \
+            -DCMAKE_CXX_COMPILER=g++-12 \
+            -DBUILD_LAGOM=ON
+        if: ${{ matrix.os == 'ubuntu-22.04' }}
+
+      - name: Create build directory macOS
+        run: |
+          # Note: We are using Apple Clang to create Universal binary
+          cmake -S Meta/Lagom -B Build -G Ninja \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" \
+            -DBUILD_LAGOM=ON
+        if: ${{ matrix.os == 'macos-12' }}
+
+      - name: Build and package js
+        working-directory: Build
+        run: |
+          ninja js
+          cpack
+
+      - name: Upload js package
+        uses: actions/upload-artifact@v3.1.1
+        with:
+          name: serenity-js-${{ matrix.package_type }}
```

Comment:
Just checking, are the packing names on this run out of date?
https://github.com/ADKaster/serenity/actions/runs/3592175050

Was going to ask if we can name the Linux artifact `serenity-js-linux-x86_64` instead of `serenity-js-ubuntu-22.04-x86_64`, but it seems like that should already be happening here?

Reply:
ah, yeah. I updated the PR to use `serenity-js-Linux-x86_64`  but didn't push that up to my fork's master branch